earthaccess/auth.py

```diff
@@ -370,6 +372,19 @@ def _persist_user_credentials(self, username: str, password: str) -> bool:
         my_netrc = Netrc(str(netrc_path))
         my_netrc["urs.earthdata.nasa.gov"] = {"login": username, "password": password}
         my_netrc.save()
+
+        # Create and write .urs_cookies file
+        urs_cookies_file = Path.home() / ".urs_cookies"
+        urs_cookies_file.write_text("")
+
+        # Create and write to .dodsrc file
+        dodsrc_file = Path.home() / ".dodsrc"
+        dodsrc_contents = f"HTTP.COOKIEJAR={urs_cookies_file}\nHTTP.NETRC={netrc_path}"
+        dodsrc_file.write_text(dodsrc_contents)
+
+        if platform.system() == "Windows":
+            shutil.copy2(dodsrc_file, Path.cwd())
```

Comment:
```suggestion
        urs_cookies_path = Path.home() / ".urs_cookies"
        if not usr_cookies_path.exists():
            urs_cookies_path.write_text("")

        # Create and write to .dodsrc file
        dodsrc_path = Path.home() / ".dodsrc"
        if not dodsrc_path.exists():
            dodsrc_contents = f"HTTP.COOKIEJAR={urs_cookies_file}\nHTTP.NETRC={netrc_path}"
            dodsrc_path.write_text(dodsrc_contents)
        
       if dodsrc_path.exists() and platform.system() == "Windows":
            shutil.copy2(dodsrc_path, Path.cwd())
            
```

Note: I changed `*_file` to `*_path` here to match the style of the rest of the file.

Reply:
Only creating them if they don't exist is also in line with OpenDAP's NASA Tutorial's `.dodsrc` setup script:
https://github.com/OPENDAP/NASA-tutorials/blob/main/tutorials/setup_dodsrc.sh

Reply:
This all looks great! Thanks for pointing out the risks, I learn something new about these prerequisite files every day!